integration_tests/dbt_project.yml

```diff
@@ -21,6 +21,11 @@ clean-targets:
   - "dbt_packages"
   - "logs"
 
+models:
+  dbt_metrics_integration_tests:
+    metric_testing_models:
+      +materialized: table
```

Comment:
Is this forcing div/0 errors to be triggered at runtime instead of test time?

Reply:
Yep! I ran into a lot of issues with Snowflake in the past where if the model was a view it would still create the object even if there was non-permissive data. So I switched to tables to ensure the object materializes.